code/src/utils/check-screen-size/useScreenSize.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+const useScreenSize = () => {
+  const [screenSize, setScreenSize] = useState({
+    width: window.innerWidth,
+    height: window.innerHeight
+  });
+
+  useEffect(() => {
+    const handleResize = () => {
+      setScreenSize({
+        width: window.innerWidth,
+        height: window.innerHeight
+      });
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    // Clean up the event listener when the component unmounts
```

Comment:
Remove comment, because it does not bring additional info. It is obvious what this function does for react developers

Reply:
Done